hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/builder/RolloutUpdate.java

```diff
@@ -30,4 +35,35 @@
      * @return updated builder instance
      */
     RolloutUpdate description(String description);
+
+    /**
+     * @param set
+     *            for {@link Rollout#getDistributionSet()}
+     * @return updated builder instance
+     */
+    default RolloutUpdate set(final DistributionSet set) {
+        return set(Optional.ofNullable(set).map(DistributionSet::getId).orElse(null));
+    }
+
+    /**
+     * @param setId
+     *            for {@link Rollout#getDistributionSet()}
+     * @return updated builder instance
+     */
+    RolloutUpdate set(long setId);
```

Comment:
The ID cannot be updated actually, right? So it shouldn't be included in the `RolloutUpdate` class